node-ipc/server.js

```diff
@@ -0,0 +1,28 @@
+var child_process = require("child_process");
+const fs = require('fs');
+
+console.log("in parent")
+
+logfile = fs.createWriteStream("cardano-node.log", { flags: "a" });
+
+logfile.on("open", function () {
+  console.log("log file opened");
+
+  const subprocess = child_process.spawn("cardano-node", [
+    "--wallet-address", "127.0.0.1:0",
+    "--no-tls",
```

Comment:
Is this file just for testing? I'm assuming Daedalus will be the server going forward. As long as it's just for testing, the no-tls is fine.

Reply:
yeah, purely for testing, daedalus should read the launcher yaml and run it with the proper params
or node reads its own yaml